pkg/conduit/runtime.go

```diff
@@ -146,6 +146,22 @@ func NewRuntime(cfg Config) (*Runtime, error) {
 
 	orc := orchestrator.NewOrchestrator(db, logger, plService, connService, procService, connPluginService, procPluginService)
 
+	var schemaService schemaregistry.Service
+	schemaService = cfg.Schema.Service
```

Comment:
@hariso I wondered about the same. I felt this could be useful, like we do for DB being able to replace the driver altogether, but I could leave that one out for now.